cms/templates/cms/home_page.html

```diff
@@ -217,31 +165,76 @@ <h3>Why should I enroll?</h3>
+            <p>Fast-track your master’s degree at MIT and  many other <a href="/pathways-graduate-programs/">institutions</a> around the world.</p>
 
           </div>
           <div class="enroll-reason col-md-3">
             <div class="reasons-images-wrap">
-              <img src="{% static 'images/cost.png' %}" alt="">
+              <img src="{% static 'images/dome_icon.webp' %}" alt="">
             </div>
-             <h4>Save Money &amp; Time</h4>
-            <p>A MicroMasters program is an affordable option for professionals who can’t take off time to earn a traditional master’s degree.</p>
+             <h4>Be part of the MIT community</h4>
+            <p>Become an affiliate member of the MIT Alumni Association and receive special access to professional journals and publications.</p>
           </div>
           <div class="enroll-reason col-md-3">
             <div class="reasons-images-wrap">
               <img src="{% static 'images/globe-icon.png' %}" alt="">
             </div>
              <h4>Grow Your Network</h4>
-            <p>After you earn your MicroMasters® program credential, you will become an affiliate of the MIT Alumni Association.{% if benefits_url %} <a href="{{ benefits_url }}">Learn More</a>{% endif %}</p>
+            <p>After you earn your MicroMasters® program credential, Build connections through a private group of like-minded professionals.</p>
+          </div>
+        </div>
+        {% if benefits_url %}
+          <div class="benefits">
+            <a href="{{ benefits_url }}">Learn more about these benefits</a>
+          </div>
+        {% endif %}
+      </section>
+
+      <section class="b2b">
+        <div class="container">
+          <img src="{% static 'images/b2b.webp' %}" alt="B2B">
+          <div class="content-holder">
+            <h3>For Organizations and Institutions</h3>
+            <p>
+              Equip your workforce and organization with new applicable skills through the MITx MicroMasters program!
+             </p> 
+             <p>
+              We work closely with organizations globally that wish to integrate our programs into their own education 
+              and training portfolios. Flexible pricing options available. 
+             </p>
+             <a class="btn btn-primary text-uppercase px-5 py-2" href="/organizations/">Inquire Today</a>
+          </div>
+        </div>
+      </section>
+
+      <section class="events">
+        <div class="container">
+          <h3>Join us at our free online events</h3>
+          <div class="content-holder">
+            <a class="img-link" href="events/"><img src="{% static 'images/events.png' %}" alt="Events"></a>
+            <a class="btn btn-primary text-uppercase px-5 py-2" href="events/">View Events</a>
+          </div>
```

Comment:
This section now has a paragraph of text, see https://projects.invisionapp.com/d/main/#/console/20687974/435513872/comments

> Register to one of our upcoming webinars to learn more about the different MicroMasters programs. Our events are hosted by the course teams, including MIT faculty, and they provide a great opportunity to ask questions about our courses